pkg/exporter/grpc_proto.go

```diff
@@ -37,7 +40,8 @@ func StartGRPCProto(hostPort string, maxFlowsPerMessage int) (*GRPCProto, error)
 // ExportFlows accepts slices of *flow.Record by its input channel, converts them
 // to *pbflow.Records instances, and submits them to the collector.
 func (g *GRPCProto) ExportFlows(input <-chan []*flow.Record) {
-	log := glog.WithField("collector", g.hostPort)
+	grpcSocket := fmt.Sprintf("[%s]:%d", g.hostIP, g.hostPort)
```

Comment:
Square brackets should only be added if g.hostIP is an IPv6 address, otherwise it should just be "%s:%d".

Reply:
this one is just debug log not used to opne grpc channel so I can use %s:%d for both